src/lib/ngsi10/UpdateContextRequest.cpp

```diff
@@ -68,16 +66,16 @@ UpdateContextRequest::UpdateContextRequest(const std::string& _contextProvider,
 *
 * UpdateContextRequest::render - 
 */
-std::string UpdateContextRequest::render(ConnectionInfo* ciP, RequestType requestType, const std::string& indent)
+std::string UpdateContextRequest::render(const std::string& apiVersion, bool asJsonObject, const std::string& indent)
 {
   std::string  out = "";
   std::string  tag = "updateContextRequest";
 
   // JSON commas:
   // Both fields are MANDATORY, so, comma after "contextElementVector"
-  //
+  //  
```

Comment:
As discussed at skype, let's consider style-check aspect in the last PR in the "refactor cycle". NTC